North-Seattle-College/ad440-winter2021-thursday-repo
# Changed GitHub action for Artillery to test the new function /users

Closes #148 

This PR adds a GitHub action that when there is a push to the **"development"** branch, will test the endpoint **"/users"** and saves the results to the **"artillery"** S3 bucket.

### To Test

1. Clone/fork my [repo](https://github.com/kevinles117/ad440-winter2021-thursday-repo/tree/development) and checkout the development branch.
2. Add a commented out line in any file so that you can commit to the development branch.
3. Head to the GitHub actions page to see if the action (Artillery action to test function users) has started
4. Wait until the process finishes.
![image](https://user-images.githubusercontent.com/46364044/108661718-6344f980-7481-11eb-8c2d-1c58e5a889cf.png)
5. Check the AWS artillery bucket to see if the json file has been generated. The file name should be in the format of artillery_YEAR_MONTH_DAY_HOURS_MINUTES_SECONDS.json
